### DISCUSSION TITLE
Class App\Filament\Resources\RelationManagers\{AnyManager} Not Found

### DISCUSSION THREAD

**Discussion by HomaEEE** [filamentphp/filament#3352] [Help] [answered]
Error after update to latest version 2.15.7 via composer 
Class "[App\Filament\Resources\RelationManagers\ProductsRelationManager](https://flareapp.io/share/17WlnMG5#context-context-versions)" not found
But this class was not even created

**Comment by ryangjchandler** [comment, marked as answer]
Can you search your codebase for references to that class please.

**Reply by HomaEEE** [reply]
<img width="210" alt="Снимок экрана 2022-08-02 в 15 46 34" src="https://user-images.githubusercontent.com/1429343/182378074-5ab0ce90-26d7-47e3-90bd-58fc64baf393.png">
Not found any files

and log from console: 
`Class "App\Filament\Resources\RelationManagers\ProductsRelationManager" not found

  at vendor/filament/filament/src/PluginServiceProvider.php:110
    106▕ 
    107▕                     continue;
    108▕                 }
    109▕ 
  ➜ 110▕                 Livewire::component($relation::getName(), $relation);
    111▕             }
    112▕ 
    113▕             foreach ($resource::getWidgets() as $widget) {
    114▕                 Livewire::component($widget::getName(), $widget);

      +9 vendor frames 
  10  [internal]:0
      Illuminate\Foundation\Application::Illuminate\Foundation\{closure}(Object(Filament\FilamentServiceProvider))

      +5 vendor frames 
  16  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))`

**Reply by danharrin** [reply]
Please run `composer dump-autoload && php artisan filament:upgrade`

**Reply by HomaEEE** [reply]
same error 
if comment 110 line in vendor/filament/filament/src/PluginServiceProvider.php - worked

**Reply by danharrin** [reply]
Please search for just `ProductsRelationManager` within your codebase, its probably in a resource's `getRelations()` method

**Reply by HomaEEE** [reply]
> Please search for just `ProductsRelationManager` within your codebase, its probably in a resource's `getRelations()` method
too much coding is bad
I found a relation where it is not even used

Tnx

**Reply by pkmasomo** [reply]
> same error if comment 110 line in vendor/filament/filament/src/PluginServiceProvider.php - worked
this worked for me too when creating stats widgets: "Class "App\Filament\Resources\Widgets\FarmerOverview"... just commenting it out  // Livewire::component($widget::getName(), $widget); helped

**Comment by zenepay** [comment]
I got the same problem, when I make laravel in a sub director : domain.com/myapp/  I don't get error if I deploy to domain.com/ . I got the issue of of livewire in cache somewhere direct to / root path of the domain which was set in local dev environment differently in production.
